content/en/synthetics/mobile_app_testing/mobile_app_tests/steps.md

```diff
@@ -172,11 +172,114 @@ If you know that a page or page element takes more than 60 seconds to load, you
 
 By default, mobile app tests wait for a page to be fully loaded before performing a step or the next step with a timeout of 60 seconds. This additional time is systematically added to **every run** of your mobile app test's recording.
 
-#### Press back
+#### Toggle Wi-Fi
 
-Interacting with the **Back** button below the mobile application records a step. Available on Android only.
+This action allows you to enable or disable Wi-Fi within your test to monitor how your application performs with or without internet access.
 
-{{< img src="mobile_app_testing/test_steps/press_back_2.mp4" alt="Recording a Press Back step in a mobile test" video=true >}}
+{{< img src="mobile_app_testing/test_steps/toggle_wifi.png" alt="Screenshot of the Toggle Wi-Fi interaction step" style="width:60%" >}}
+
+#### HTTP requests
+
+You can run HTTP requests as part of your mobile app tests. If you are using [restrictive networks][8], ensure the required IP address ranges are added to your to allowlist to enable HTTP steps to run in your Synthetic Monitoring Mobile tests.
+
+**Note**: You can copy an HTTP step from a Synthetic Monitoring Mobile test and re-use it into a Synthetic Monitoring Browser test. However, copying steps from a Browser test to a Mobile test is not supported.
+
+{{< img src="mobile_app_testing/test_steps/http_interaction.png" alt="Mobile app testing interactions Run HTTP test" style="width:70%;" >}}
+
+To define your HTTP request:
+
+1. Select **Interaction**, then **Run HTTP Test**. Enter the URL you wish to test.
```

Comment:
```suggestion
1. Enter the URL you wish to test.
```
Same note as above with these first clicks are kind of already covered 🙂

Reply:
fixed!